activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -6,7 +6,9 @@ module ActiveRecord
   module FinderMethods
     ONE_AS_ONE = "1 AS one"
 
-    # Find by id - This can either be a specific id (1), a list of ids (1, 5, 6), or an array of ids ([5, 6, 10]).
+    # Find by id - This can either be a specific id (ID), a list of ids (ID, ID, ID), or an array of ids ([ID, ID, ID]).
+    # `ID` here has a meaning of an "identifier" which for models with a single-column primary key will be a single value
+    # and for models with a composite primary key will be an array of values.
```

Comment:
```suggestion
    # `ID` refers to an "identifier". For models with a single-column primary key, `ID` will be a single value,
    # and for models with a composite primary key, it will be an array of values.
```